product_docs/docs/biganimal/release/administering_cluster/projects.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Managing projects
+description: "Use projects to provide additional security and segregation to workflows within your organization"
+---
+
+Projects are a group of database clusters within a BigAnimal organization. Access to projects is controlled by a user's role. With projects, you can separate workflows to provide secure and isolated environments and assign different users to different projects. See [Managing portal access](/biganimal/release/administering_cluster/01_portal_access) for more information.
+
+Projects actions are available from the left navigation bar. The actions apply to the project in focus. You can switch between projects using the Projects menu.
+
+## Adding a user to a project
+Before adding a user to a project: 
+- The user must be added through your organization's identify provider.
+
+- The user must log in to the BigAnimal portal at least once.
+
+To add a user: 
+1. Select the project you want to add a user to from the Projects menu.
+
+1. Select Users from the left navigation bar.
+2.  Select the edit icon for the user.
+3.  Select **Assign Roles**.
+4.  Depending on the level of access you want for the user, select the appropriate role.
+
+5.  Select **Submit**.
+
+!!! Note
+    For a user's role assignment to take effect, the user must log out from BigAnimal and log in again.
+
+## Creating a project
+
+Before creating a new project: 
+- You must create a new Azure subscription or AWS account to create a new project. 
+
+  - To create a new Azure subscription, see [Create a Microsoft Customer Agreement subscription](https://learn.microsoft.com/en-us/azure/cost-management-billing/manage/create-subscription).
+
+  - To create a new AWS account, see [Creating an AWS account in your organization](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_accounts_create.html).
```

Comment:
we need to mention about one restriction explicitly: 

the same Azure subscription or AWS account can only be used to register for one project at the same time,e.g. I have 2 projects, and I only need to deploy my clusters in AWS, then I will have to create 2 different AWS account. 

If the user want to reuse the cloud account (azure subscription or aws account), the user has to delete the project that was previously connected with that account at first.

Reply:
fixed